Speech/tests/Snippet/V1/SpeechClientTest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Tests\Snippets\Speech\V1;
+
+use Google\ApiCore\BidiStream;
+use Google\ApiCore\Call;
+use Google\ApiCore\Testing\MockBidiStreamingCall;
+use Google\ApiCore\Transport\TransportInterface;
+use Google\Cloud\Core\Testing\Snippet\SnippetTestCase;
+use Google\Cloud\Speech\V1\LongRunningRecognizeResponse;
```

Comment:
After removing the partials for recognize/long running recognize I think we forgot to clean up the imports in the tests  here. Looks like there are some unused ones through this and other test files.